page/style.css

```diff
@@ -1,223 +1,607 @@
-  position: relative;
-  float: left;
-}
-article:nth-child(7) > figure > img {
-  width: 561px;
-  margin-left: -90px;
-  margin-top: -119px;
-  z-index: -1;
-  position: relative;
-}
-article:nth-child(7) > section {
-  width: 485px;
-  font-family: merriweather;
-  font-weight: 200;
-  font-size: 13px;
-}
-article:nth-child(7) > h1 {
-  font-family: oswald;
-  font-size: 55px;
-  font-weight: 200;
-  border-bottom: 12px solid black;
-  width: 471px;
-  margin-top: -31px;
-}
-article:nth-child(7) > h3 {
-  font-family: oswald;
-  margin-top: -45px;
-  margin-bottom: 14px;
-  font-size: 25.8px;
-  border-bottom: 3px solid black;
-  width: 471px;
-  line-height: 67px;
-}
-article:nth-child(8) {
-
-}
-article:nth-child(8) {
-
-}
-article:nth-child(8) {
-
-}
-article:nth-child(8) {
+	font-family: lato;
+	font-weight: 100;
+	color: white;
+	font-size: 143px;
+	margin-left: 130px;
+	display: inline-block;
+	line-height: .8;
+	z-index: 5;
+	width: 50px;
+	margin-bottom: 158px;
+	margin-top: 10px;
+	position: relative;
+}
+
+.box-model {
+	width: 331px;
+	background-color: #cdc9dd;
+	margin: -445px 0 -28px 23px;
+	height: 637px;
+	padding-bottom: 16px;
+	padding-right: 15px;
+}
+
+.box-model img {
+	width: 276px;
+	margin: 230px 0px -7px 9px;
+	z-index: 6;
+	position: relative;
+}
+
+.box-model h1 {
+	margin-left: 152px;
+	font-family: lora;
+	font-style: italic;
+	font-weight: bold;
+	color: #5f5d66;
+	line-height: 2;
+	margin-top: -15px;
+}
+
+.box-model section {
+	margin-top: -24px;
+	margin-left: 90px;
+	width: 234px;
+	font-family: lato;
+	text-align: center;
+	color: #6e6c77;
+	font-size: 13px;
+	line-height: 1.5;
+	font-weight: 500;
+}
+
+.positioning-relative {
+	position: absolute;
+	top: 63px;
+	left: 271px;
+}
+
+.positioning-relative > h1 {
+	color: #fdcd84;
+}
+
+.positioning-relative > p {
+	color: #fdcd84;
+}
+
+.positioning-relative p {
+	margin-top: -138px;
+	margin-left: 149px;
+	z-index: 10;
+	font-size: 12px;
+	letter-spacing: 2px;
+}
+
+.positioning-relative h1 {
+	font-family: oswald;
+	margin-left: 149px;
+	letter-spacing: 2px;
+	font-size: 35px;
+	width: 30px;
+	margin-top: 7px;
+	line-height: 1.2;
+}
+
+.positioning-relative > a > figure > img {
+	height: 400px;
+	margin-left: 7px;
+	margin-top: -17px;
+}
+
+.positioning-relative > section {
+	font-family: merriweather;
+	font-weight: 300;
+	width: 400px;
+	margin-left: 149px;
+	margin-top: -17px;
+}
+
+.staying-afloat {
+	float: left;
+	margin-top: 97px;
+	margin-left: 107px;
+}
+
+.staying-afloat > h1 {
+	color: #247a5a;
+	font-family: oswald;
+	margin-bottom: 5px;
+	font-size: 18px;
+	width: 115px;
+	text-align: right;
+}
+
+.staying-afloat > section {
+	font-family: lato;
+	font-size: 11.5px;
+	line-height: 1.3;
+	text-align: right;
+	width: 115px;
+}
+
+.staying-afloat > a > figure > img {
+	height: 125px;
+	margin-left: -36px;
+}
+
+.list-style {
+	margin-top: 109px;
+	float: left;
+	margin-bottom: -91px;
+}
+
+.bullet_header {
+	border-top: 1px solid lightgray;
```

Comment:
Not a fan of non-specific color codes. In the comp there are specific color specifications.